tests/TestCase.php

```diff
@@ -9,15 +9,36 @@ class TestCase extends \PHPUnit_Framework_TestCase
     protected $_skinCode;
     protected $_hmacSignature;
 
+	/**
+	 * Settings parsed from the config/test.ini file
+	 *
+	 * @var array
+	 */
+    protected $settings;
+
     public function __construct()
     {
-
         $this->_merchantAccount = $this->getMerchantAccount();
         $this->_skinCode = $this->getSkinCode();
         $this->_hmacSignature = $this->getHmacSignature();
+		$this->settings = $this->_loadConfigIni();
 
+		$this->setDefaultsDuringDevelopment();
     }
 
+	/**
+	 * During development of the standalone php api library this function sets the necessary defaults which would
+	 * otherwise set by the merchant
+	 */
+    private function setDefaultsDuringDevelopment()
+	{
+		if ($this->isDev()) {
```

Comment:
what's the downside on skipping the dev flag and isDev for these unit tests?

Reply:
Right now nothing because the only setting we apply is the default time zone and only if the client doesn't have a default value for that, but for the future I'd say this flag will be useful
If you say that do it more agile and skip the parts that's not really necessary for us now I can remove it

Reply:
I believe is better to remove for simplicity. Unit tests are mostly considered dev env anyway.